includes/options/class-amp-options-manager.php

```diff
@@ -623,6 +624,41 @@ public static function show_response_cache_disabled_notice() {
 		);
 	}
 
+
+	/**
+	 * Outputs an admin notice if the site is not served over HTTPS.
+	 *
+	 * @since 1.3
+	 *
+	 * @return void
+	 */
+	public static function insecure_connection_notice() {
+		// is_ssl() only tells us whether the admin backend uses HTTPS here, so we add a few more sanity checks.
+		$uses_ssl = (
+			is_ssl()
+			&&
+			( strpos( get_bloginfo( 'wpurl' ), 'https' ) !== 0 )
```

Comment:
Shouldn't this be `===`? 🤔

Reply:
Oh, yes, indeed. Good catch, that's what I get for copying from the site health code... :)

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3219/commits/243320364c10ea0e0e79af0e6a36900a15a6f7ea

Reply:
TIL: `get_bloginfo( 'wpurl' )` and `get_bloginfo( 'url' )` are wrappers around `site_url()` and `home_url()` respectively (which are themselves wrappers around `get_option()`, 🐢🐢🐢).

Maybe it would be preferable to do `'https' !=== wp_parse_url( get_bloginfo( 'wpurl' ), PHP_URL_SCHEME )` here and below? A tiny bit more robust perhaps.